### DISCUSSION TITLE
Appending GUI

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7625] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

Is there a GUI that use ImageMagick appending images vertically and horizontally?

**Comment by fmw42** [comment]
The online web interface at https://magickstudio.imagemagick.org/scripts/MagickStudio.cgi

**Reply by Jay123210599** [reply]
How would that work?
